pkg/model/config/config.go

```diff
@@ -104,6 +93,28 @@ func (config *Config) AddResource(gvk GVK) bool {
 	return true
 }
 
+// ValidGroup checks that the provided group can be added to the project
+func (config Config) ValidGroup(group string) bool {
+	// Only v2 is single-group by default and requires a flag to allow multi-group, short-circuit the rest
+	if config.Version != Version2 {
```

Comment:
I would prefer we use the centralized implementation instead of duplicate it here. ( IsV1/IsV2)

Reply:
I did it for any `config.Version == VersionX` check.